server/controller/handler/chart.go

```diff
@@ -66,6 +66,7 @@ type Spec struct {
 	Experiments     []string `yaml:"experiments"`
 	ChaosExpCRDLink string   `yaml:"chaosexpcrdlink"`
 	Platforms       []string `yaml:"platforms"`
+	ChaosType  		string `yaml:"chaosType"`
```

Comment:
Fix the indentation

Reply:
in my code base already indented

Reply:
Can you check tab-space ratio in your editor?